release/ray_release/config.py

```diff
@@ -40,11 +40,13 @@
 
 
 def read_and_validate_release_test_collection(
-    config_file: str, schema_file: Optional[str] = None
+    config_files: List[str], schema_file: Optional[str] = None
 ) -> List[Test]:
     """Read and validate test collection from config file"""
```

Comment:
no unit tests that requires update?

Reply:
there's only one in the test_config.py file